src/style.css

```
@@ -0,0 +1,89 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300px;
+}
+
+.search__form {
+  margin-top: 20px;
+}
+
+.container {
+  display: flex;
+  position: relative;
+}
+
+.logo {
+  position: absolute;
+  left: 26px;
+  bottom: 26px;
+  top: 25px;
+}
+
+.logo-small {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  bottom: 16px;
+  top: 15px;
+}
```

```suggestion
.logo {
  position: absolute;
  left: 26px;
  bottom: 26px;
  top: 25px;
}

.logo-small {
  width: 11px;
  height: 11px;
  left: 13px;
  bottom: 16px;
  top: 15px;
}
```
Don't repeat the same styles